src/tests/test_dynamic_api/test_views_mvt.py

```diff
@@ -0,0 +1,94 @@
+import mapbox_vector_tile
+import pytest
+from schematools.contrib.django import models
+
+from dso_api.dynamic_api.permissions import fetch_scopes_for_dataset_table, fetch_scopes_for_model
+
+
+@pytest.fixture(autouse=True)
+def clear_caches():
+    yield  # run tests first
+    fetch_scopes_for_dataset_table.cache_clear()
+    fetch_scopes_for_model.cache_clear()
```

Comment:
Supergoede ontdekking! Wat mij betreft komt deze in `conftest.py` als default voor alle tests!

Reply:
Copy-paste van test_views_wfs.py :)

Ik laat het verplaatsen naar conftest graag aan jou, aangezien je toch de tests al aan het verbouwen bent.

Reply:
oh hahaha :-D